### ISSUE TITLE
chapter2.4.3 momentum issue

### ISSUE THREAD

**Issue by sunjiaxin111** [fchollet/deep-learning-with-python-notebooks#19]
this is a naive implementation of gradient descent with momentum,
and velocity = past_velocity * momentum + learning_rate * gradient,
but i think it should be velocity = past_velocity * momentum - learning_rate * gradient
